.github/workflows/ci-test-hosted.yml

```
@@ -252,6 +258,17 @@
           working-directory: app/client
           env: "NODE_ENV=development"
 
+      - name: Stop Monitoring
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:1:6: Quote this to prevent word splitting

(shellcheck)